var/spack/repos/builtin/packages/libristra/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Libristra(CMakePackage):
+    """ The ristra library is a set of support utilities for
+    ristra codes, including simple mathematical operations, physical
+    units, and input parsing
+    """
+    homepage = 'https://github.com/laristra/libristra'
+    url = 'https://github.com/laristra/libristra/archive/master.zip'
+    git = 'https://github.com/laristra/libristra.git'
+    tags = ['ristra']
+
+    version('master', branch='master', submodules=False, preferred=True)
+    version('1.0.0', commit='33235fe0334ca7f1f99b386a90932d9f8e1e71de')
+
+    variant('build_type', default='Release', values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'),
+            description='The build type to build', multi=False)
+    variant('paraview', default=False,
+            description='Enable ParaView')
+    variant('shared_lua', default=False,
+            description='Build with shared lua')
```

Comment:
Minor OCD (not a request at all): would these two directive fit a single line each?

Reply:
The previous better maps to what we do internally for the more elaborate spackages, but changed to better match upstream conventions